lib/index.js

```diff
@@ -701,15 +701,21 @@ MapboxGeocoder.prototype = {
   /**
    * Set input
    * @param {string} searchInput location name or other search input
+   * @param {boolean} [showSuggestions=false] display suggestion on setInput call
    * @returns {MapboxGeocoder} this
    */
-  setInput: function(searchInput) {
+  setInput: function(searchInput, showSuggestions) {
+    if (showSuggestions === undefined) {
+      showSuggestions = false
+    }
```

Comment:
```suggestion
  setInput: function(searchInput, showSuggestions = false) {
```

Reply:
I tried this at first but it is not possible due to uglify version as you mentioned bellow
I don't think that upgrading the version in this PR is useful.